fix: Add /FORCEREGISTRY flag to windows installer (#5517)
grafana/alloy: docs/sources/set-up/install/windows.md

- `/DISABLEREPORTING=<yes|no>` Disable [data collection][]. Default: `no`
- `/DISABLEPROFILING=<yes|no>` Disable profiling endpoint. Default: `no`
- `/ENVIRONMENT="KEY=VALUE\0KEY2=VALUE2"` Define environment variables for Windows Service. Default: ``

> - `/FORCEREGISTRY=yes` Force the installer to write install options (for example, `/STABILITY`, `/DISABLEREPORTING`) to the Windows registry even when upgrading over an existing installation. By default, existing registry values are left unchanged on upgrade, so new options passed during an upgrade are not applied unless you use this flag.

Reviewer: ```suggestion
- `/FORCEREGISTRY=yes` Force the installer to write install options such as `/STABILITY` and `/DISABLEREPORTING` to the Windows registry even when upgrading.
  By default, the installer preserves registry values on upgrade, so options passed during an upgrade aren't applied unless you use this flag.
```
